internal/guest/runtime/hcsv2/network.go

```diff
@@ -216,9 +215,7 @@ func (n *namespace) Sync(ctx context.Context) (err error) {
 
 	if n.pid != 0 {
 		for i, a := range n.nics {
-			// Lower the metric for anything but the first adapter
-			// TODO: remove when we correctly support assigning metrics to the default GWs
-			if i > 0 {
+			if a.adapter.PolicyBasedRouting && i > 0 {
 				a.adapter.EnableLowMetric = true
```

Comment:
can this be, so that we're resilient to the uVM accidentally sending a bad/malformed request?
```suggestion
			if a.adapter.PolicyBasedRouting {
				a.adapter.EnableLowMetric = (i > 0)
```

Reply:
Done